sqlglot/dialects/sqlite.py

```diff
@@ -62,6 +62,21 @@ def _transform_create(expression: exp.Expression) -> exp.Expression:
     return expression
 
 
+def _parse_json_extract(args: t.List) -> exp.JSONExtract:
+    this, *paths = args
+    path = parser.parse_json_path(seq_get(paths, 0))
+    if isinstance(path, exp.JSONPath):
+        for p in paths[1:]:
+            parsed = parser.parse_json_path(p)
```

Comment:
what's up with this sub parser?

Reply:
Sqlite supports multiple json paths in `JSON_EXTRACT`, i.e. `JSON_EXTRACT(x, '$.x', '$.y', ...)`. This parses each one of them and then merges them into a single AST node. The `paths[1:]` slice is taken so that we skip the root node in all subsequent paths.

Reply:
interesting

Reply:
I'll double-check this section because it may be incorrect, noticed a similar weirdness with [MySQL](https://github.com/tobymao/sqlglot/pull/2883/files#diff-14ef9d8b99f48dcf8f6a4e3bbab7bf55a9d85d2b83d684c3c822f0f94fcd1d07R152-R155) and refactored what I initially implemented there too.

Reply:
I got rid of this after all because SQLite and MySQL semantics match when there are >1 paths, see https://github.com/tobymao/sqlglot/pull/2883/commits/9d5f9f5a188fb904834e5a12a8cbe09990f60827. Thanks for calling this out!